packages/react-native-editor/__device-tests__/helpers/test-data.js

```diff
@@ -2,7 +2,7 @@ exports.shortText = `Rock music approaches at high velocity.`;
 
 exports.mediumText = `The finer continuum interprets the polynomial rabbit. When can the geology runs? An astronomer runs. Should a communist consent?`;
 
-exports.longText = `Beneath the busy continuum blinks the ineffective husband. Why a metric now outside the official subway? How can the prompt crop exhaust his tree 
+exports.longText = `Beneath the busy continuum blinks the ineffective husband. Why a metric now outside the official subway? How can the prompt crop exhaust his tree
```

Comment:
Hmm just wondering if this will not break other tests where this `longText` is used :)

Reply:
It shouldn't, but just to be sure will revert that change.